master/buildbot/worker/base.py

```diff
@@ -461,6 +461,18 @@ def attached(self, conn):
             # reasonable fallback
             self.path_module = namedModule("posixpath")
         log.msg("bot attached")
+
+        #LLVM_LOCAL begin
+        log.msg(f"  host: {workerinfo['host']}")
+        log.msg(f"  admin: {workerinfo['admin']}")
+        log.msg(f"  access_uri: {workerinfo['access_uri']}")
+        log.msg(f"  version: {workerinfo['version']}")
+        log.msg(f"  system: {self.worker_system}")
+        log.msg(f"  worker_commands: {self.worker_commands}")
+        log.msg(f"  basedir: {self.worker_basedir}")
+        log.msg(f"  environ: {self.worker_environ}")
```

Comment:
Forgot to mention. Since we print out multiple lines one by one, the printout could be interleaved by other log messages from different threads. Maybe prepare the whole thing first, and then print it once? Would be much easier to read and track in the log.